File: template/python3/function/handler.py
# -*- coding: utf-8 -*-
from flask import Response
import json

def handle(req):
    """handle a request to the function
    Args:
        req (str): request body
    """

    data = {
        'hello'  : 'world',
        'number' : 3
    }
    js = json.dumps(data)

    resp = Response(js, status=300, mimetype='application/json')
    resp.headers['Authorization'] = 'Bearer abc123'

    return resp

File: template/php/function/src/Handler.php
<?php
namespace App;

require ('Response.php');
use ResponseModel\Resp;

/**
 * Class Handler
 * @package App
 */
class Handler
{
    /**
     * @return
     */
    public function handle($request) {
        //$response->StatusCode = 503;
        //$response->Body = "Hello, world!";
        $response = new Resp();
        $response->Body = "Hello, world!!";
        return $response;
    }
}

?>